### ISSUE TITLE
Telemetry ApplicationInsights error

### ISSUE THREAD

**Issue by Janghou** [marlonfan/coc-phpls#34]
Great extension, but I'm getting a `ApplicationInsights:CorrelationIdManager [ { Error: connect ECONNREFUSED 0.0.0.0:443 `every time I edit a php file.
Also initially high CPU.

1. open any php file
2. :CocInfo

**Desktop (please complete the following information):**
Kubuntu 18.04
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Jun 06 2019 17:31:41)
Included patches: 1-1453

What is the use of Application Insights. Telemetry?
Can it be disabled or removed?

**Comment by marlonfan** [comment]
this process not belongs to coc-phpls

**Comment by Janghou** [comment]
To what process then?

:CocInfo

```
## Output channel: phpls

[Info  - 10:44:23 AM] Initialising intelephense 1.2.3
[Info  - 10:44:23 AM] Reading state from ~/.config/coc/extensions/coc-phpls-data/5ec608e0.
ApplicationInsights:CorrelationIdManager [
  Error: connect ECONNREFUSED 0.0.0.0:443
      at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1056:14) {
    errno: 'ECONNREFUSED',
    code: 'ECONNREFUSED',
    syscall: 'connect',
    address: '0.0.0.0',
    port: 443
  }
]
[Info  - 10:44:23 AM] Initialised in 152 ms
[Info  - 10:44:23 AM] Intelephense 1.2.3
[Info  - 10:44:23 AM] Indexing started.
[Info  - 10:44:23 AM] Indexing ended. 1 files indexed in 0s.
[Info  - 10:44:23 AM] Writing state to ~/.config/coc/extensions/coc-phpls-data/5ec608e0.
[Info  - 10:44:23 AM] Wrote state in 0.2s.
```

**Comment by marlonfan** [comment]
Let me check. In addition, I suggest you upgrade the version of coc-phpls

**Comment by Janghou** [comment]
Shouldn't that be done automatically?

> You don't need to update coc extensions manually, coc detects acceptable new version of installed extension everyday (by default) the first time it starts. When it finds a new version of an extension, it will update it for you automatically.

https://github.com/neoclide/coc.nvim/wiki/Using-coc-extensions

Otherwise I have to file another bug.

**Comment by marlonfan** [comment]
coc-phpls latest version is 2.1.2, your version's 1.2.3. it's old version

**Comment by Janghou** [comment]
Yes 2.1.2 is the version I use. Just checked that with :CocList.

The error relates to intelephense 1.2.3, which I suppose is something that is used in  coc-phpls

I guess:
https://github.com/bmewburn/vscode-intelephense

There is no intelephense 2.1.2.
